client/src/pages/Dashboard.js

```diff
@@ -184,45 +169,100 @@ function UserInfo(props) {
 function ProjectCard(props) {
   const classes = useStyles();
   const projectInfo = props.project;
+  const fromNow = moment(projectInfo.deadline).fromNow();
+
   return (
     <Grid item xs={6}>
       <Card elevation={8}>
         <CardMedia
           className={classes.media}
           component="img"
-          src={projectInfo.photo}
+          src={
+            projectInfo.photos[0]
+              ? "https://plphotos.s3.us-east-2.amazonaws.com/" +
```

Comment:
might want to store the amazon s3 url in an env var so you can easily change it if needed

Reply:
Good idea